samza-core/src/test/scala/org/apache/samza/system/TestSystemConsumers.scala

```diff
@@ -347,32 +348,15 @@ class TestSystemConsumers {
     val systemStreamPartition1 = new SystemStreamPartition(system, stream, new Partition(1))
     val systemStreamPartition2 = new SystemStreamPartition(system, stream, new Partition(2))
 
-    val consumer = new TestStartpointConsumer
```

Comment:
Please rename the test, since there is no visitor anymore.

Reply:
Done.